<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowsParametersDialog</class>
 <widget class="QDialog" name="WindowsParametersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры окон расчета атрибутов</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/imgs/calculator.png</normaloff>:/icons/imgs/calculator.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="nbWindowsLabel">
       <property name="text">
        <string>Количество окон расчета атрибутов</string>
       </property>
       <property name="buddy">
        <cstring>nbWindowsSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="nbWindowsSpinBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="ratioGroupBox">
       <property name="minimumSize">
        <size>
         <width>415</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Расчет Соотношений</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="ratio1ComboBox"/>
          </item>
          <item>
           <widget class="QLabel" name="devideLabel">
            <property name="text">
             <string>/</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ratio2comboBox"/>
          </item>
          <item>
           <widget class="QLabel" name="minRatioLabel">
            <property name="text">
             <string>Мин. Соотн.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="minRatioLineEdit">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="addRatioPushButton">
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="2">
         <widget class="QTableWidget" name="relationsTableWidget">
          <property name="minimumSize">
           <size>
            <width>290</width>
            <height>0</height>
           </size>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>160</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Выборка трасс</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Минимальная
амплитуда</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="minAmplDoubleSpinBox">
            <property name="decimals">
             <number>7</number>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.000001000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkTestsCheckBox">
            <property name="text">
             <string>Учитывать тесты</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="mutedCheckBox">
            <property name="text">
             <string>Не учитывать
отключенные трассы</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="rmsSpectrumCheckBox">
         <property name="text">
          <string>Среднеквадратический спектр</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="energyGroupBox">
         <property name="title">
          <string>Параметры энергии спектра</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="minFreqLabel">
            <property name="text">
             <string>Начальная частота</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="minFreqSpinBox">
            <property name="maximum">
             <number>250</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="maxFreqLabel">
            <property name="text">
             <string>Конечная частота</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="maxFreqSpinBox">
            <property name="maximum">
             <number>250</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="widthGroupBox">
         <property name="title">
          <string>Параметры ширины спектра</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0">
           <widget class="QLabel" name="frqLvlLabel">
            <property name="text">
             <string>Уровень измерения, %</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QRadioButton" name="energyRadioButton">
            <property name="text">
             <string>Относительно энергии сигнала</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="frqLvlSpinBox"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QRadioButton" name="maxFreqRadioButton">
            <property name="text">
             <string>Относительно пиковой частоты</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTableWidget" name="windowsTableWidget"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WindowsParametersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WindowsParametersDialog</receiver>
   <slot>saveSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>813</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>651</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nbWindowsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WindowsParametersDialog</receiver>
   <slot>nbWindowsSpinBoxValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addRatioPushButton</sender>
   <signal>clicked()</signal>
   <receiver>WindowsParametersDialog</receiver>
   <slot>addRatioSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxFreqRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>frqLvlSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveSettings()</slot>
  <slot>nbWindowsSpinBoxValueChanged(int)</slot>
  <slot>windowsTableWidgetItemChanged(QTableWidgetItem*)</slot>
  <slot>addRatioSlot()</slot>
  <slot>displayMenu()</slot>
 </slots>
</ui>
